pkg/agent/agent.go

```diff
@@ -103,6 +97,12 @@ func (k *Klocksmith) process(stop <-chan struct{}) error {
 		return err
 	}
 
+	// we are schedulable now.
+	glog.Info("Marking node as schedulable")
+	if err := k8sutil.Unschedulable(k.nc, k.node, false); err != nil {
```

Comment:
Other than #122 I haven't seen any other reference to this specific issue, although we have some other issues with being good neighbors (#50, #61).